tests/components/myuplink/test_init_2.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for init module - alternate fixtures."""
+from unittest.mock import MagicMock
+
+import pytest
+
+from homeassistant.components.myuplink.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+@pytest.fixture
+def load_systems_file() -> str:
+    """Load fixture file for systems."""
+    return load_fixture("systems-2dev.json", DOMAIN)
+
+
+@pytest.fixture
+def load_device_file() -> str:
+    """Fixture for loading device file."""
+    return load_fixture("device-2dev.json", DOMAIN)
+
+
+async def test_devices_multiple_created_count(
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "load_systems_file",
    [load_fixture("systems-2dev.json", DOMAIN)],
)
async def test_devices_multiple_created_count(
```

Reply:
Why don't we use this test fixture as default?

Reply:
I had the same thought the other day. It is a good time to swap the fixtures now.